subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -49,27 +49,21 @@ This is a potentially breaking change if you are consuming the console output of
 
 === Deprecations
 
-[[custom_configuration_roles]]
```

Comment:
If we are getting rid of this section, we should also get rid of the deprecation logger which references it: https://github.com/gradle/gradle/blob/master/subprojects/core/src/main/java/org/gradle/api/internal/artifacts/configurations/ConfigurationRole.java#L77-L84

We always set `warnOnCustomRole` to false anyways, so its not like the code ever got called. I think this would be easy to reintroduce when we have ways for users to declare their own roles

Reply:
That's unnecessary for now, since that code will never run, and it's not an error to reference a section that doesn't exist.  I'll note it for 8.2 cleanup.